lib/test/iso/bmff/labl.test.ts

```diff
@@ -4,6 +4,7 @@ describe('labl box', function () {
 	it('should correctly parse the box from sample data', function () {
 		const box = findBox('SRMP_AC4.mp4', [meta, prsl, labl])
 			.boxes?.filter((box: Box) => box.type === 'grpl')[0]
+			// @ts-expect-error: Add type for `boxes` and `groupId` in `grpl` box
```

Comment:
No idea how to properly fix it, I didn't see any types related to `grpl` box, hence, this comment